src/octokit/get-probot-octokit-with-defaults.ts

```diff
@@ -32,11 +34,13 @@ export function getProbotOctokitWithDefaults(options: Options) {
   const authOptions = options.githubToken
     ? {
         token: options.githubToken,
+        request: options.request,
```

Comment:
patch options.request everywhere. I have the feeling that it should be checked, that we pass everywhere where applicable

Reply:
We need to pass this for mocking?

Reply:
Yes. There can also cases we dont test in probot, but because we maybe dont pass the request.fetch option, a developer can maybe not test while developing a probot app because we forgot to pass the request.fetch option to the corresponding plugin.

Before with nock we did not needed to be that explicit.